src/openforms/js/components/admin/form_design/logic/actions/synchronize_children/SynchronizeChildrenConfigModal.js

```diff
@@ -0,0 +1,196 @@
+import {Form, Formik, useFormikContext} from 'formik';
+import {useContext} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {FormContext} from 'components/admin/form_design/Context';
+import {getComponentDatatype} from 'components/admin/form_design/variables/utils';
+import {makeNewVariableFromComponent} from 'components/admin/form_design/variables/utils';
+import Field from 'components/admin/forms/Field';
+import Fieldset from 'components/admin/forms/Fieldset';
+import FormRow from 'components/admin/forms/FormRow';
+import Select from 'components/admin/forms/Select';
+import {
+  ValidationErrorContext,
+  ValidationErrorsProvider,
+} from 'components/admin/forms/ValidationErrors';
+import VariableMapping from 'components/admin/forms/VariableMapping';
+import {FAIcon} from 'components/admin/icons';
+import ErrorMessage from 'components/errors/ErrorMessage';
+
+const ManageChildrenMappings = () => {
```

Comment:
@viktorvanwijk apart from processing all your comments I have updated/refactored the component (after @sergei-maertens suggestion) in order to be similar with DMN configuration. Now we have variable-property mappings and a nicer UI with generic `childrenMappings` errors and for individual rows.